101-media-services-create/README.md

```diff
@@ -0,0 +1,9 @@
+# Create a Media Services Account using a template
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fazure%2Fazure-quickstart-templates%2Fmaster%2F101-media-services-create%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+
+This template creates a Media Services Account with its Storage account on Azure.
+
+For more information about Azure Media Services, see [Media Services Documentation](https://docs.microsoft.com/en-us/azure/media-services/).
```

Comment:
@mathieu-benoit It's totally up to you, as you already have a link which tells what is media services and etc. But I think it will be nicer to have a line or two explaining what resources will be deployed and how you should use that media service resource from portal. Then for further detail you can point to the link you have given. Again, this is what I think, you can ignore this :)